src/contribute/release-guide.md

```diff
@@ -158,12 +169,54 @@ Adjust any of the above properties to the improve clarity and presentation of th
 
 ### Verify that a Release Build Works
 
-Run `mvn -Prelease clean install` to ensure that the build processes that are specific to that
-profile are in good shape.
+Pre-installation for python build
+* Install pip
+  
+  ```
+  curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+  python get-pip.py
+  ```
+* Install virtualenv
+
+  ```
+  pip install --upgrade virtualenv
+  ```
+* Cython
+
+  ```
+  sudo pip install cython
+  sudo apt-get install gcc
+  sudo apt-get install python-dev
+  ```
+Make sure your ```time``` alias to ```/usr/bin/time```, if not:
```

Comment:
Why do we need ``time`` for release verification?

Reply:
python tests script uses ```time```.  If ```/usr/bin/time``` is not alias to ```time```, tests will fail.